contributors/design-proposals/node/troubleshoot-running-pods.md

```diff
@@ -730,7 +730,7 @@ coupling it with container images.
 *   [CRI Tracking Issue](https://issues.k8s.io/28789)
 *   [CRI: expose optional runtime features](https://issues.k8s.io/32803)
 *   [Resource QoS in
-    Kubernetes](https://git.k8s.io/kubernetes/docs/design/resource-qos.md)
+    Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/)
```

Comment:
How about link to this [design proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/resource-qos.md)?

Reply:
Sure. Thanks @ScorpioCPH